testing/test_hsm.py

```diff
@@ -48,6 +48,52 @@
             'tb1q0pu8s7rc0pu8s7rc0pu8s7rc0pu8s7rc0pu8s7rc0pu8s7rc0puq3mvnhv',
 ]
 
+def compute_policy_hash(policy):
+    # Computes a policy hash for comparison purposes. This has to do some conversion and cleanup
+    # in order to be able to see a policy the same way the firmware sees it.
+    hasher = sha256()
+    top_keys = ['must_log', 'never_log', 'priv_over_ux', 'warnings_ok', 'msg_paths',
+                    'share_xpubs', 'share_addrs', 'notes', 'period', 'allow_sl', 'set_sl', 'boot_to_hsm']
+    for key in top_keys:
+        if value := policy.get(key):
```

Comment:
`:=` this forks off all < py3.8 from running tests (I personally do not care as I already use py3.8 but have no idea about "others")

Reply:
yeah that's ok, keep it